src/keysym.c

```diff
@@ -64,7 +64,7 @@ get_name(const struct name_keysym *entry)
 XKB_EXPORT int
 xkb_keysym_get_name(xkb_keysym_t ks, char *buffer, size_t size)
 {
-    if ((ks & ((unsigned long) ~0x1fffffff)) != 0) {
+    if ((ks & ((unsigned long) ~XKB_MAX_KEYSYM)) != 0) {
```

Comment:
This looks a bit weird, e.g. if the max was 535353 then it wouldn't have made sense to apply `~` to it. With a MAX constant it's clearer to do `if (ks > XKB_MAX_KEYSYM) { ... }`.

Reply:
Yes. I went the lazy road to keep diff minimal.